core/user-flow.js

```diff
@@ -82,22 +82,33 @@ class UserFlow {
    * @return {LH.UserFlow.StepFlags}
    */
   _getNextNavigationFlags(flags) {
-    const nextFlags = this._getNextFlags(flags) || {};
+    const newStepFlags = this._getNextFlags(flags) || {};
 
-    if (nextFlags.skipAboutBlank === undefined) {
-      nextFlags.skipAboutBlank = true;
+    if (newStepFlags.skipAboutBlank === undefined) {
+      newStepFlags.skipAboutBlank = true;
     }
 
+    // BFCache will actively load the page in navigation mode.
+    // To prevent this from impacting future flow steps, we can disable the audit.
+    // If `skipAudits` is defined on the config, we should append this rather than override.
+    const skipAuditsSet = new Set(
+      newStepFlags.skipAudits ||
+      this._options?.config?.settings?.skipAudits ||
```

Comment:
Overriding how config and flags merge out here definitely feels like an architecture failure. Should we be using a different mechanism for this?

Reply:
Yes. We should have some way for gatherers to determine that they are in a user flow so that `BFCacheFailures` can prevent the active gathering.

I was thinking about a new setting flag like `usePassiveGathering` which could be applicable to other gatherers like the screenshot which can behave more passively.

Reply:
I like that idea. I guess the bfcache audit could mark itself n/a if the gatherer ran passively since there's not really a passive fallback? New [navigation, timespan, snapshot] x [passive] permutations aren't great for implementation/testing, but ideally there won't be any difference between any of those modes for almost all gatherers

Reply:
Sounds good, but that's probably a separate PR

Reply:
yes, definitely :)